src/containers/Trips/MulticityTrip.js

```diff
@@ -61,10 +61,9 @@ export class MulticityTrip extends Component {
   componentDidUpdate(prevProps) {
     const { history } = this.props;
     const { multicityTripData } = this.props;
-    if (prevProps.multicityTripData !== multicityTripData) {
-      if (multicityTripData.message === 'Trip requested successfully') {
-        return history.push('/requests');
-      }
+    const isTripChanged = prevProps.multicityTripData !== multicityTripData;
```

Comment:
this line is unnecessary. No need to hold this in a variable.

Reply:
I thought writing in one condition would make the line big. I removed the variable and broke condition in multiple lines.